sites/backend/tests/shared.mjs

```diff
@@ -1,6 +1,6 @@
 import dotenv from 'dotenv'
 import axios from 'axios'
-import chai from 'chai'
+import * as chai from 'chai'
```

Comment:
I don't think it will be easy to avoid importing the entire `chai` namespace.

In `sites/backend/tests/shared.mjs`, the `setup()` method stores `chai` and also returns it:
```js
export const setup = async () => {
  const store = {
    chai,
  ...
  }
  return { chai, config, expect, store } 
 } 
```

`sites/backend/tests/index.mjs` passes along `chai` to runTests() which passes it along to the individual tests:
```js
 const runTests = async (...params) => {
   await userTests(...params)
   await mfaTests(...params)
   await apikeyTests(...params)
   await accountTests(...params)
   await setTests(...params)
   await bookmarkTests(...params)
   await curatedSetTests(...params)
   await optionPackTests(...params)
   await patternTests(...params)
   await subscriberTests(...params)
   await flowTests(...params)
 }
 
 // Load initial data required for tests
 const { chai, config, expect, store } = await setup()
 
 // Note run the tests using this data
 runTests(chai, config, expect, store)
```

And, individual tests use multiple methods from `chai`. For example, in `sites/backend/tests/account.mjs`:
```js
export const accountTests = async (chai, config, expect, store) => {
 ...
          chai
             .request(config.api)
             .patch(`/account/${auth}`)
             .set(
               'Authorization',
               auth === 'jwt'
                 ? 'Bearer ' + store.account.token
                 : 'Basic ' +
                     new Buffer(
                       `${store.account.apikey.key}:${store.account.apikey.secret}`
                     ).toString('base64')
             )
             .send(body)
             .end((err, res) => {
               expect(err === null).to.equal(true)
...
```

Reply:
Sure thing.